packages/api/src/controllers/v1/users/mod.rs

```diff
@@ -127,8 +151,8 @@ impl UserController {
             .await?;
         tx.commit().await?;
 
-        let jwt = self.generate_token(&user)?;
-        Ok(JsonWithHeaders::new(user).with_bearer_token(&jwt))
+        let jwt = AppClaims::generate_token(&user)?;
+        Ok(JsonWithHeaders::new(user).with_auth_cookie("Bearer", &jwt))
```

Comment:
```suggestion
        Ok(JsonWithHeaders::new(user).with_auth_cookie(AuthCookie::Bearer, &jwt))
```
이런식으로 구성하는게 좋지 않을까 싶네요.